cmd/launcher/svc_windows.go

```diff
@@ -232,3 +249,75 @@ func (w *winSvc) Execute(args []string, r <-chan svc.ChangeRequest, changes chan
 		}
 	}
 }
+
+// determineRootDirectory is used specifically for windows deployments to override the
+// configured root directory if another one containing a launcher DB already exists
+func determineRootDirectory(opts *launcher.Options, slogger *slog.Logger) string {
```

Comment:
```suggestion
func determineRootDirectory(slogger *slog.Logger, opts *launcher.Option) string {
```

Reply:
👍 will do in a separate commit so I can update the caller too